lib/corpus.py

```diff
@@ -330,16 +341,16 @@ def __init__(self):
         super().__init__()
         self.start = 0.0
         self.end = 0.0
-        self.track = None
-        self.orth = None
-        self.speaker_name = None
+        self.track = None  # type: Optional[int]
+        self.orth = None  # type: Optional[str]
+        self.speaker_name = None  # type: Optional[str]
 
-        self.recording = None
+        self.recording = None  # type: Optional[Recording]
 
-    def fullname(self):
+    def fullname(self) -> str:
         return self.recording.fullname() + "/" + self.name
 
-    def speaker(self):
+    def speaker(self) -> Speaker:
         return self.recording.speaker(self.speaker_name)
 
     def dump(self, out, indentation=""):
```

Comment:
you missed a few functions 👋

Reply:
I did not miss them, I was too lazy to look up the type for out :) 

But okay, as I got caught I will search for it.

Reply:
`typing.TextIO` ?
https://docs.python.org/3/library/typing.html#typing.IO

Reply:
It should be all `TextIO` right?